src/components/Pages/TeamsPage/TeamsPage.js

```diff
@@ -189,15 +192,17 @@ class TeamsPage extends React.Component {
                   className="col-3"
                   style={{ paddingRight: "10px", maxWidth: "20%" }}
                 >
-                  <MEButton
-                    style={{ width: "100%", margin: 0 }}
-                    onClick={() => {
-                      this.setState({ createTeamModalOpen: true });
-                    }}
-                    className="phone-vanish"
-                  >
-                    Start Team
-                  </MEButton>
+                  {userType === STANDARD_USER && (
```

Comment:
Even though this works, try to make "isStandardUser" a property on the whole user object in redux. Otherwise we would have to recreate this same logic in so many places when we have to implement standard-user-specific functionality in future. This way, we can just use "user.isStandardUser" anywhere freely. And in the case that the logic for detecting standard users changes, we only need to change in one place.

Reply:
done